### PR TITLE
move to sphinx

### PR DESCRIPTION
**Pull request by lynxis**
@mwarning what do you thing?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,287 +32,9 @@ Specific features of Nodogsplash are configurable, by editing the configuration
...
-  will also mark packets and use IMQ's. Therefore one or both of Nodogsplash and
-  a QOS package may malfunction if used together. Potential conflicts may be
-  investigated by looking at your overall iptables setup. To check to see all
-  the rules in, for example, the mangle table chains, run
-
-    ```iptables -t mangle -v -n -L```
-
-  For extensive suggestions on debugging iptables, see for example Oskar
-  Andreasson's_tutorial.
+For additonal documentation please look at https://nodogsplash.readthedocs.io/en/latest/
```
Comment (mwarning) [inline README.md]: additonal => additional

**Comment by mwarning** [comment]
@lynxis: it is important to me, that the documentation originates in this repository and in human readable/textual form.
I'm fine with this changes, but not a big fan of adding outside dependencies for the documentation. :p

**Comment by lynxis** [comment]
@mwarning there are .rst files under docs/. these used for generation.

**Comment by lynxis** [comment]
everything but the travis part is merged.
for travis testing we need https://github.com/travis-ci/apt-package-whitelist/issues/3660 so we can change to trusty.

**Pull request closed without merging**